openerp/openupgrade/doc/source/analyse.rst

```diff
@@ -1,29 +1,27 @@
-How to run your own analysis
-============================
-If you do need to run your own analysis, you need to perform the following
-steps (the awkward processing of the server log file is now obsolete).
-
-* Set up two OpenUpgrade servers of subsequent Odoo releases
-
-* On both instances, install a database without demo data and
-  install the *openupgrade_records* module, which is included in the
-  OpenUpgrade server distribution. This will add a menu
-  *OpenUpgrade Development* to the Administration menu.
-
-* On both instances, install the modules that you need to write migration
-  scripts for, or alternatively select *Install All Modules* from the
-  Development menu.
-
-* On both instances: from the development menu, select the *Generate Records*
-  option.
-
-* On the target instance (this is the more recent version): from the
-  Development menu, select the *Comparison Config* option and
-  create a new config to connect to the other instance. In the config's
-  form, click on *Perform Analysis*.
-
-Note that in many of the operations above you may get a client timeout or a
-concurrent access error even if the operation completes successfully. You
-should be able to assertain a succesful operation by verifying that all
-modules involved are in an installed state and the analysis files in the
-module directories have an appropriate modification time.
+Run your own analysis
+=====================
+
+1. First you need to prepare your Python environment by installing the appropriate release of openupgradelib_.
+
+2. In order to analyse the difference of two codebases you need to specially prepare your different codebases by applying this OpenUpgrade patch onto your Odoo code-base. (Note@ProjectLeader: Please provide patches towards OCA/OCB and standard Odoo for different versions) This will add some folders of the OpenUpgrade tool suite and modify some Odoo core files in order to work smoothly with OpenUpgrade.
+
```

Comment:
2015-09-04 22:24 GMT-05:00 David Arnold <notifications@github.com>:

> > +2. In order to analyse the difference of two codebases you need to
> specially prepare your different codebases by applying this OpenUpgrade
> patch onto your Odoo code-base. (Note@ProjectLeader: Please provide
> patches towards OCA/OCB and standard Odoo for different versions) This will
> add some folders of the OpenUpgrade tool suite and modify some Odoo core
> files in order to work smoothly with OpenUpgrade.


OpenUpgrade is a "Use One Time" distribution, You do not need to apply any
patch on any place, simply run and export the migrated db.


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>